react/components/EXPERIMENTAL_TableTree/README.md

```diff
@@ -194,14 +194,16 @@ const columns = [
 const items = sampleData
 
 function Image({ size }) {
+  const style = {
+    width: '100%',
+    height: '100%',
```

Comment:
Could those two elements being replaced by `w-100` and `h-100` with css classes?

Reply:
yes, it could